phpBB/phpbb/db/migration/tool/module.php

```diff
@@ -81,6 +81,9 @@ public function get_name()
 	*/
 	public function exists($class, $parent, $module)
 	{
+		// Clear the Modules Cache first to get actual data
+		$this->cache->destroy("_modules_$class");
```

Comment:
@VSEphpbb @rxu method `destroy` exists for each cache driver and works OK because of [this](https://github.com/rxu/phpbb3/blob/0cc092073405166a0da72009823256a00389d0b8/phpBB/phpbb/cache/service.php#L94-L97).
Probably IDEs cannot handle this case at the moment.